checkpoint/cp-resource-management/README.md

```diff
@@ -1,23 +1,30 @@
-#  Dome9 asset management agent
+#  Check Point Cloudguard Dome9 agents
 
 ## Introduction
 
-This chart creates a single resource management Pod that scans the cluster's resources (Pods, Services, Nodes, PSP, Network Policy, and Ingress) and uploads them to [Dome9](https://secure.dome9.com/).
-Dome9 provides compliance, Visibility, Monitoring and Threat Hunting capabilities.
+This chart creates a single resource management Pod that scans the cluster's resources (Nodes, Images, Pods, Namespaces, Services, PSP, Network Policy, and Ingress) and uploads them to [Check Point ClougGuard Dome9](https://secure.dome9.com/).
+Check Point ClougGuard Dome9 provides Compliance, Vulnerability Assessment, Visibility, Monitoring and Threat Hunting capabilities.
 
 ## Prerequisites
 
 - Kubernetes 1.12+
 - Helm 3.0+
-- A Dome9 account and API key
+- A Check Point ClougGuard Dome9 account and API key
 
 ## Installing the Chart
 
 To install the chart with the chosen release name (e.g. `my-release`), run:
 
 ```bash
 $ helm repo add checkpoint https://raw.githubusercontent.com/CheckPointSW/charts/master/repository/
-$ helm install my-release checkpoint/cp-resource-management --set-string credentials.user=[Dome9 API Key] --set-string credentials.secret=[Dome9 API Secret] --set-string clusterID=[Dome9 Cluster ID]
+$ helm install my-release checkpoint/cp-resource-management --set-string credentials.user=[CloudGuard Dome9 API Key] --set-string credentials.secret=[CloudGuard Dome9 API Secret] --set-string clusterID=[Dome9 Cluster ID]

```

Comment:
[minor] 'Dome9 Cluster ID' --> 'CloudGuardDome9 Cluster ID'

Reply:
Actually we need to change CloudGuard Dome9 to CloudGaurd across the board